app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -15,12 +17,12 @@ class SubmitForm526
 
       # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
       sidekiq_retries_exhausted do |msg, _ex|
-        transaction_class.update_transaction(msg['jid'], :exhausted)
-        log_message_to_sentry(
-          "Failed all retries on Form526 submit, last error: #{msg['error_message']}",
-          :error
+        TRANSACTION_CLASS.update_transaction(msg['jid'], :exhausted)
+        Rails.logger.send(
```

Comment:
can this be:
```suggestion
        Rails.logger.error('Form526 Exhausted', 'job_id' => msg['jid'], 'error_message' => msg['error_message'])
```
This matches how the tracking module logs. The Rails.logger is not the standard one we use a [structured logger](https://rocketjob.github.io/semantic_logger/) so you can append key values to a log line. This makes finding the log line by message easier and then the extra data lives in a payload object in CloudWatch so you can use its [filter syntax](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html).